cmd/create/operatorroles/cmd.go

```diff
@@ -41,6 +41,7 @@ var modes []string = []string{"auto", "manual"}
 var args struct {
 	clusterKey          string
 	prefix              string
+	operatorRolesPrefix string
```

Comment:
We should find a way to remove that `prefix` since the policies should be automatically assigned from tags, and then make the `--operator-roles-prefix` become the new `--prefix`

Reply:
@vkareh  can you please clarify what tags you are mentioning?

Reply:
@vkareh  updated to find the prefix from the tag and then use it to find the policy. PTAL

Reply:
We still need to re-purpose the existing `--prefix` to behave like `--operator-roles-prefix`. This is the flow I have in mind:
1. User runs `rosa create operator-roles --cluster foo --prefix bar`
2. ROSA creates roles `bar-openshift-machine-api-etc...`
3. ROSA fetches the corresponding policy by tag
4. ROSA attaches the policy to the role

Reply:
If we use the prefix from the `cluster.AWS().STS().RoleARN()`, then step 3 here would be purely by string. Still need to check that the policy itself matches (possibly through tags). If not, there was an error in the `rosa create account-roles` command.